.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+# Description of changes:
+
+# CTSM Issues Fixed (include github issue #):
+
+# Are answers expected to change? (and if so in what way):
+
+# Any User Interface Changes (namelist or namelist defaults changes)?:
+
+# Testing performed:
+(List what testing you did to show your changes worked as expected)
+(This can be manual testing or running of the different test suites)
+
+# NOTE: Be sure to check your Coding style against the standard:
```

Comment:
I like this better (having a note but not expecting them to write "yes") - thanks.